templates/zerver/help/search-for-messages.md

```diff
@@ -54,7 +54,7 @@ shared history](/help/stream-permissions), including messages sent
 before you joined the stream (or organization) or those sent to public
 streams you are not subscribed to.
 
-By default, Zulip searches messages in your personal history, i.e. the
+By default, Zulip searches messages in your your history, i.e. the
```

Comment:
The word `your` is repeated twice here?

Reply:
Made the required changes.

Reply:
Hi, that looks better. Can you also remove the extra space before `your`?

You should also change your commit message according to the bot's comment. You can use `git commit --amend` to change the commit message of your last commit.
Then simply push it to the PR with `git push -f`.